e2e/nomostest/config-sync.go

```diff
@@ -1287,9 +1287,10 @@ func resetRootRepos(nt *NT, upstream string, sourceFormat filesystem.SourceForma
 			resetRootRepo(nt, name, nt.RootRepos[name].UpstreamRepoURL, nt.RootRepos[name].Format)
 			nt.WaitForSync(kinds.RootSyncV1Beta1(), name, configsync.ControllerNamespace,
 				nt.DefaultWaitTimeout, DefaultRootSha1Fn, RootSyncHasStatusSyncCommit, nil)
-			// Now the repo is back to the initial commit, we can delete the safety check namespace
+			// Now the repo is back to the initial commit, we can delete the safety check Namespace & ClusterRole
 			nt.RootRepos[name].Remove(nt.RootRepos[name].SafetyNSPath)
-			nt.RootRepos[name].CommitAndPush("delete the safety check namespace")
+			nt.RootRepos[name].Remove(nt.RootRepos[name].SafetyClusterRolePath)
+			nt.RootRepos[name].CommitAndPush("delete the safety check namespace & role")
```

Comment:
commit message might as well say ClusterRole for clarity when reading the test output

Reply:
done